lib/policy_machine/policy_element.rb

```diff
@@ -230,14 +231,26 @@ def allowed_assignee_classes
     end
   end
 
-  # An operation set in a policy machine.
+  # An operation in a policy machine.
   class Operation < PolicyElement
-    def self.create(unique_identifier, policy_machine_uuid, pm_storage_adapter, extra_attributes = {})
+    def self.create(unique_identifier, policy_machine_uuid, pm_storage_adapter, extra_attributes = {}, prohibition = false)
+      if unique_identifier =~ /^~/ && !prohibition
+        raise ArgumentError, "An operation cannot start with '~'"
```

Comment:
Shouldn't this more clearly be <code>A non prohibited operation cannot start...</code>